examples/demo-stf/README.md

```diff
@@ -0,0 +1,99 @@
+## Enabling RPC via SDK Macros
```

Comment:
@preston-evans98 is also working on big `README` for this crate: https://github.com/Sovereign-Labs/sovereign/pull/321/files

can you please make sure this part  fits the flow in the other readme?